index.html

```diff
@@ -59,21 +73,104 @@ <h1 class="work-main-heading">My Recent Works</h1>
         <hr class="divider" />
         <div class="main-card">
           <div class="post-image"></div>
-          <h4 class="work-title-post">Multi-Post Stores</h4>
-          <p class="work-supporting-text">
+          <div class="main-card-content">
+            <h4 class="work-title-post">Multi-Post Stores</h4>
+            <p class="work-supporting-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="languages-list">
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <div class="btn-left">
```

Comment:
- [ ] This project card is not showing on your User interface. Kindly fix it
![image](https://user-images.githubusercontent.com/63473584/178101367-6227577d-7603-41d9-9600-51162ab096b4.png)

Reply:
Hi @vickymarz, I don't know why it is not showing on your User interface. On my laptop and I also check for the other devices it's working fine here.